src/side-nav/side-nav.jsx

```diff
@@ -0,0 +1,370 @@
+    this.setState({ open: true });
+    if (this.props.onNavOpen) this.props.onNavOpen();
+    return this;
+  },
+
+  getThemePalette() {
+    return this.context.muiTheme.palette;
+  },
+
+  getTheme() {
+    if(this.context.muiTheme.component.sideNav)
+      return this.context.muiTheme.component.sideNav;
+    else
+      return {
+        width: 256,
+        backgroundColor: '#FFFFFF',
+      };
+  },
+
+  getStyles() {
+    let x = this._getTranslateMultiplier() * (this.state.open ? 0 : this._getMaxTranslateX());
+    let styles = {
+      root: {
+        height: '100%',
+        width: this.props.width? this.props.width : this.getTheme().width,
+        backgroundColor: this.props.backgroundColor? this.props.backgroundColor : this.getTheme().backgroundColor,
+        position: 'fixed',
+        zIndex: 10,
+        left: 0,
+        top: 0,
+        transform: 'translate3d(' + x + 'px, 0, 0)',
+        transition: !this.state.swiping && Transitions.easeOut(),
+        overflow: 'hidden',
+      },
+      menu: {
+        backgroundColor: this.props.backgroundColor? this.props.backgroundColor : this.getTheme().backgroundColor,
+        overflowY: 'auto',
+        overflowX: 'hidden',
+        height: '100%',
+        width: this.props.width? this.props.width : this.getTheme().width,
+        borderRadius: '0',
+      },
+      list: {
+        width: this.props.width? this.props.width : this.getTheme().width,
+        paddingTop: 0,
+      },
+      rootWhenOpenRight: {
+        left: 'auto',
+        right: 0,
+      },
+    };
+
+    return styles;
+  },
+
+  render() {
+    let overlay;
+
+    let {
+      className,
+      openType,
+      defaultOpen,
+      header,
+      onChange,
+      onNavOpen,
+      onNavClose,
+      openRight,
+      disableSwipeToOpen,
+      style,
+      listStyle,
+      menuStyle,
+      zDepth,
+      ...other,
+    } = this.props;
+
+    let styles = this.getStyles();
+    if (openType==='overlay') {
+      overlay = (
+        <Overlay
+          ref="overlay"
+          show={this.state.open || !!this.state.swiping}
+          transitionEnabled={!this.state.swiping}
+          onTouchTap={this._onOverlayTouchTap}
+        />
+      );
+    }
+
+    return (
+      <div className={className}>
+        {overlay}
+        <Paper
+          ref="container"
+          className="paper"
+          zDepth={2}
+          rounded={false}
+          transitionEnabled={!this.state.swiping}
+          style={this.mergeAndPrefix(
+            styles.root,
+            openRight && styles.rootWhenOpenRight,
+            style)}>
+            {header}
+            <Menu
+              {...other}
+              ref="menu"
+              autoWidth={false}
+              style={this.mergeAndPrefix(styles.menu,menuStyle)}
+              listStyle={this.mergeAndPrefix(styles.list,listStyle)}
+              zDepth={zDepth}
+              onItemTouchTap={this._onMenuItemClick} >
+              {this.props.children}
```

Comment:
That seems the only line different from LeftNav (aside importing another Menu compnent),